OpenEdXMobile/src/main/java/org/edx/mobile/event/AppUpdatedEvent.java

```diff
@@ -0,0 +1,22 @@
+package org.edx.mobile.event;
+
+import android.support.annotation.NonNull;
+
+public class AppUpdatedEvent {
+    @NonNull
+    public final long previousVersionCode;
+    @NonNull
+    public final long currVersionCode;
```

Comment:
`curVersionCode` would be a better name here, to have similarity with the `curVersionName` variable below.